src/main/java/org/eclipse/xpanse/terraform/boot/models/response/TerraformResult.java

```diff
@@ -36,4 +33,6 @@ public class TerraformResult {
     @Schema(description = "Data of all other files generated by the terraform execution."
             + "The map key contains the file name and value is the file contents as string.")
     private Map<String, String> importantFileContentMap;
+    @Schema(description = "Terraform with version which executed the command.")
```

Comment:
terraformVersionUsed
The version of the Terraform binary used to execute scripts